go/src/socialapi/workers/payment/subscription.go

```diff
@@ -119,30 +116,45 @@ func EnsureSubscriptionForGroup(groupName string, params *stripe.SubParams) (*st
 		return nil, ErrCustomerNotExists
 	}
 
+	hasSource, err := checkCustomerHasSource(group.Payment.Customer.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	if !hasSource {
+		return nil, ErrCustomerSourceNotExists
```

Comment:
The `checkCustomerHasSource` could return `ErrCustomerSourceNotExists` if count == 0 and nil otherwise, so no need for `checkCustomerHasSource` could just have 1 return value.